Cubical/Data/Sigma/Properties.agda

```diff
@@ -160,23 +78,6 @@ discreteΣ {B = B} Adis Bdis (a0 , b0) (a1 , b1) = discreteΣ' (Adis a0 a1)
         ... | (no ¬q) = no (λ r → ¬q (subst (λ X → PathP (λ i → B (X i)) b0 b1) (Discrete→isSet Adis a0 a0 (cong fst r) refl) (cong snd r)))
     discreteΣ' (no ¬p) = no (λ r → ¬p (cong fst r))
 
-Σ-contractFst : ∀ {ℓ ℓ'} {A : Type ℓ} {B : A → Type ℓ'} (c : isContr A)
-  → Σ A B ≃ B (c .fst)
-Σ-contractFst {B = B} c =
-  isoToEquiv
-    (iso
-      (λ {(a , b) → subst B (sym (c .snd a)) b})
-      (c .fst ,_)
-      (λ b →
-        cong (λ p → subst B p b) (isProp→isSet (isContr→isProp c) _ _ _ _)
-        ∙ transportRefl _)
-      (λ {(a , b) →
-        sigmaPath→pathSigma _ _ (c .snd a , transportTransport⁻ (cong B (c .snd a)) _)}))
-
--- a special case of the above
-ΣUnit : ∀ {ℓ} (A : Unit → Type ℓ) → Σ Unit A ≃ A tt
-ΣUnit A = isoToEquiv (iso snd (λ { x → (tt , x) }) (λ _ → refl) (λ _ → refl))
-
 assocΣ : (Σ[ (a , b) ∈ Σ A B ] C a b) ≃ (Σ[ a ∈ A ] Σ[ b ∈ B a ] C a b)
```

Comment:
More naming: shouldn't this be \Sigma-assoc or something match the earlier lemmas in the file?